bin/src/main.rs

```diff
@@ -143,7 +153,7 @@ async fn main() {
             }
 
             let k8s_event_stream = match config.log.log_k8s_events {
-                K8sTrackingConf::Never => None,
+                K8sTrackingConf::Never => None, // Agent won't release it this is set to None.
```

Comment:
Believe there's a typo here: `it => if`

Reply:
Good catch. I removed this comment as it no longer applies.